docs/list.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sapota UI | Typography</title>
+    <link rel="icon" href="./assets/sapota-icon.png" type="image/x-icon" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link rel="stylesheet" href="./css/doc.css" />
+    <link rel="stylesheet" href="../style.css" />
+  </head>
+  <body>
+    <header class="top-nav box-shadow main-header">
+      <nav class="navigation">
+        <div class="logo-btn-header">
+          <a href="../index.html" class="nav-brand">SAPOTA UI</a>
+          <button class="menu-icon">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
```

Comment:
this can be removed i guess

Reply:
No, It is for the menu button in the mobile view